<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceDialog</class>
 <widget class="QDialog" name="AppearanceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appearance</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource.qrc">
    <normaloff>:/icons/images/color-swatches.png</normaloff>:/icons/images/color-swatches.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>521</width>
     <height>361</height>
    </rect>
   </property>
   <widget class="QWidget" name="tabColors">
    <attribute name="title">
     <string>Colors</string>
    </attribute>
    <widget class="QGroupBox" name="groupColor">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>141</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Color</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="editColor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>85</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>#FFFFFF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonColor">
      <property name="geometry">
       <rect>
        <x>101</x>
        <y>151</y>
        <width>28</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button800000">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>49</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #800000; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button00FF00">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>79</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #00FF00; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button008080">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>19</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #008080; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonC0C0C0">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>49</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #C0C0C0; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button800080">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>49</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #800080; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonFF0000">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>109</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : #FF0000; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button0000FF">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>79</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0000FF; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button008000">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>19</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #008000; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button00FFFF">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>79</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #00FFFF; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonFF00FF">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>109</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FF00FF; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button808000">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>49</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #808000; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button808080">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>79</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #808080; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonFFFF00">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>109</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFF00; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button000000">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>19</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #000000; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button000080">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>19</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #000080; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonFFFFFF">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>109</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBackgroundColor">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>200</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Background Color</string>
     </property>
     <widget class="QLineEdit" name="editBackgroundColor">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>89</y>
        <width>85</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>#FFFFFF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackgroundColor">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>28</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackground00FF00">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>49</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #00FF00; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackgroundFFFF00">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>49</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFF00; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackgroundFFFFFF">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>19</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackgroundFF0000">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>49</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FF0000; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackgroundNone">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>19</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #C0C0C0; border : 2px solid black</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackground000000">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>19</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #000000; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackground00FFFF">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>49</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : #00FFFF; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBackgroundC0C0C0">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>19</y>
        <width>32</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #C0C0C0; border : 2px solid black</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupProperties">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Properties</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QListWidget" name="listColorNames">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>321</width>
        <height>281</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </widget>
    <zorder>groupBackgroundColor</zorder>
    <zorder>groupColor</zorder>
    <zorder>groupProperties</zorder>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>listColorNames</tabstop>
  <tabstop>button000000</tabstop>
  <tabstop>button000080</tabstop>
  <tabstop>button008000</tabstop>
  <tabstop>button008080</tabstop>
  <tabstop>button800000</tabstop>
  <tabstop>button800080</tabstop>
  <tabstop>button808000</tabstop>
  <tabstop>buttonC0C0C0</tabstop>
  <tabstop>button808080</tabstop>
  <tabstop>button0000FF</tabstop>
  <tabstop>button00FF00</tabstop>
  <tabstop>button00FFFF</tabstop>
  <tabstop>buttonFF0000</tabstop>
  <tabstop>buttonFF00FF</tabstop>
  <tabstop>buttonFFFF00</tabstop>
  <tabstop>buttonFFFFFF</tabstop>
  <tabstop>editColor</tabstop>
  <tabstop>buttonColor</tabstop>
  <tabstop>buttonBackgroundNone</tabstop>
  <tabstop>buttonBackground000000</tabstop>
  <tabstop>buttonBackgroundC0C0C0</tabstop>
  <tabstop>buttonBackgroundFFFFFF</tabstop>
  <tabstop>buttonBackground00FFFF</tabstop>
  <tabstop>buttonBackground00FF00</tabstop>
  <tabstop>buttonBackgroundFF0000</tabstop>
  <tabstop>buttonBackgroundFFFF00</tabstop>
  <tabstop>editBackgroundColor</tabstop>
  <tabstop>buttonBackgroundColor</tabstop>
  <tabstop>buttonSave</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>AppearanceDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked(bool)</signal>
   <receiver>AppearanceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
